cypress/e2e/featured-products.cy.js

```diff
@@ -0,0 +1,37 @@
+describe('Featured Products Section', () => {
+
+  Cypress.Commands.add('checkRating', (name) => {
+    cy.get('.product-card').find(`img[alt="${name}"]`).click()
+    .get('.product-rating').contains('5')
+    .get('.top-nav').find('img[alt="Makeup 360 logo"]').click()
```

Comment:
Can we make it so that this test starts with clicking on the logo instead of ending with it? Keeping it the way it is would mean that the second it block ends with a click. I tested it on my branch and it still works with the order switched.

Reply:
Good catch, Laura!